tests/virt_autotest/login_console.pm

```diff
@@ -128,7 +128,15 @@ sub login_to_console {
             send_key 'down';
             send_key 'ret';
             #wait sshd up
-            assert_screen('sshd-server-started', 180);
+            my $remote_machine = get_required_var('SUT_IP');
+            record_info("Will detect whether machine $remote_machine boots up and has ssh port open", "Calling virt_autotest::utils::check_port_state");
+            if (check_port_state($remote_machine, 22, 10)) {
+                record_info("$remote_machine tcp port 22 is open", "SSH server should be up for opening yast2 for offline upgrade on $remote_machine");
+            }
+            else {
+                record_info("$remote_machine tcp port 22 is not open", "SSH server is not up for opening yast2 for offline upgrade on $remote_machine");
+                die "Can not connect to machine $remote_machine to perform offline upgrade via ssh";
+            }
```

Comment:
It should be enough to just have the "die" message which also shows up on the thumbnails. No need for the additional `record_info`. As the die can abort the execution flow I would even recommend to change the test code like this

```suggestion
            die "Can not connect to machine $remote_machine to perform offline upgrade via ssh"
              unless check_port_state(get_required_var('SUT_IP'), 22, 10);
```

Reply:
Done